packages/ag-charts-community/src/module/optionModules.ts

```diff
@@ -1,6 +1,14 @@
+import type { AgCartesianSeriesOptions } from '../options/series/cartesian/cartesianSeriesTypes';
+import type { AgHierarchySeriesOptions } from '../options/series/hierarchy/hierarchyOptions';
+import type { AgPolarSeriesOptions } from '../options/series/polar/polarOptions';
 import type { BaseModule, ModuleInstance } from './baseModule';
 import type { AxisContext, ModuleContextWithParent, SeriesContext } from './moduleContext';
 
+type RemoveUndefined<T> = T extends undefined ? never : T;
```

Comment:
Nit: Try the inbuilt `NonNullable`?

Reply:
Thanks, I didn't know about `NonNullable` but that's exactly what I was looking for.